extension/war/src/main/webapp/WEB-INF/conf/social-extension/social/core-configuration.xml

```diff
@@ -730,6 +730,7 @@
             <property name="exo.feature.StreamFilter.enabled" value="${exo.feature.StreamFilter.enabled:false}" />
             <property name="exo.feature.SpaceWebNotifications.enabled" value="${exo.feature.StreamFilter.enabled:true}" />
             <property name="exo.feature.publisherRolePromotion.enabled" value="${exo.feature.publisherRolePromotion.enabled:false}" />
+            <property name="exo.feature.navigationPortlet.enabled" value="${exo.feature.navigationPortlet.enabled:false}" />
```

Comment:
should be deleted, see decision of product committee here https://github.com/Meeds-io/MIPs/issues/36#issuecomment-1375711342

Reply:
ok, thanks